business/checkers/virtual_service_checker.go

```diff
@@ -45,7 +47,7 @@ func (in VirtualServiceChecker) runGroupChecks() models.IstioValidations {
 	validations := models.IstioValidations{}
 
 	enabledCheckers := []GroupChecker{
-		virtual_services.SingleHostChecker{Namespace: in.Namespace, Namespaces: in.Namespaces, VirtualServices: in.VirtualServices},
+		virtual_services.SingleHostChecker{Namespace: in.Namespace, Namespaces: in.Namespaces, VirtualServices: append(in.VirtualServices, in.ExportedVirtualServices...)},
```

Comment:
In this case, I guess that we should go case by case into each checker to see whether we can append both sets of VS. We should understand the nature of the validation and see if it makes any sense.
I'd say that most of them would be completely alright to merge VS, but again, we should go case by case and most definitely adding test cases to make sure all is green.

Reply:
The two previous comments apply to all the ObjectCheckers (DR, VS, SE, GW, SC, etc.). I know, it is pretty tough, but we should be super accurate with this.